### PR TITLE
CreateTrip feature and Cards Display 

### PR DESCRIPTION
**Pull request by Ishaan21**

### CODE REVIEW

**Comment by ankurshaswat** [comment]
Addresses #22

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -21,6 +21,8 @@ jobs:
       # Disabled for now, causing issues
       # - run: flutter test
       - run: flutter build apk
+<<<<<<< HEAD
+=======
```
Comment (tmibvishal) [inline .github/workflows/main.yml:25]: fix this conflict. leave it as master and do not change this file

File: lib/screens/dashboard.dart [hunk 2]
```diff
@@ -1,31 +1,75 @@
...
-        ),
+      body: Column(
+        children: <Widget>[
+          Container(
+            margin: EdgeInsets.all(5),
+            height: (MediaQuery.of(context).size.height - MediaQuery.of(context).padding.top)*0.84 ,
+            width: double.infinity,
+            child: TripsList(_allRequests),
+          ),
+        ],
```
Comment (tmibvishal) [inline lib/screens/dashboard.dart:67]: Dashboard screen is showing 
```
A RenderFlex overflowed by 16 pixels on the bottom.
I/flutter (19991): The relevant error-causing widget was:
I/flutter (19991):   Column
I/flutter (19991):   file:///Users/vishal/Desktop/Coding%20B/Flutter/workspace/shareacab/lib/screens/dashboard.dart:59:13
```
Run in the emulator to check this

File: .github/workflows/main.yml (hunk 1 above)
Comment (Ishaan21) [reply .github/workflows/main.yml]: updated origin master

File: lib/screens/dashboard.dart (hunk 2 above)
Comment (arpit-saxena) [reply lib/screens/dashboard.dart]: @Ishaan21 Please look into this issue and fix it

**Comment by tmibvishal** [comment]
@Ishaan21 Please look into this issue and fix it. Also It would be nice if you can move the bottom floating screen to a new screen. Because it is generally not a good idea to keep text box in bottom floating screen as they pop up the keyboard. Also we will later add much more complex options in add group like map support and more. So I think it would be nice to have a different screen for it.

**Pull request merged**